packages/svelte/src/escaping.js

```diff
@@ -0,0 +1,26 @@
+const ATTR_REGEX = /[&"<]/g;
```

Comment:
I think it would be good to also have a test for the fact that we're now escaping `&` in attributes.

Reply:
Added